cms/urls.py

```diff
@@ -75,13 +75,13 @@
     url(r'^signin$', 'login_page', name='login'),
     url(r'^request_course_creator$', 'request_course_creator'),
 
-    url(r'^course_team/{}/(?P<email>.+)?$'.format(COURSELIKE_KEY_PATTERN), 'course_team_handler'),
+    url(r'^course_team/{}(?:/(?P<email>.+))?$'.format(COURSELIKE_KEY_PATTERN), 'course_team_handler'),
```

Comment:
@cahrens it's a non-capturing regex group. It tells regex engine that this group should not be remembered, i.e. it is only to apply "or" or quantity modifiers (`+`,`*`,`?`, `{}`). As a result, this group is not sent to django view.